test/MUXControls.Test/Infra/Application.cs

```diff
@@ -367,7 +367,7 @@ public void GoBack()
                     }
 
                     Log.Comment("Invoking the back button...");
-                    FindElement.ById<Button>("__GoBackInvoker").InvokeAndWait();
+                    FindElement.ById<Button>("__GoBackInvoker").InvokeAndWait(TimeSpan.FromSeconds(10));
```

Comment:
10 seconds... why? Include a comment here.

Reply:
There is no significance to the 10 seconds. Previously it would wait indefinitely, so if anything went wrong, the test execution could not recover. This change allows the retry logic to kick in if needed.